src/connected/connectionsetup.ts

```diff
@@ -181,7 +182,7 @@ function renderConnectionSetupPanel(context: vscode.ExtensionContext, webview: v
         <vscode-checkbox id="enableNotifications" ${!initialState.disableNotifications ? 'checked' : ''}>
           Receive notifications from ${serverProductName}
         </vscode-checkbox>
-        <input type="hidden" id="enableNotifications-initial" value="${!initialState.disableNotifications}" />
+        <input type="hidden" id="enableNotifications-initial" value="${!(initialState.disableNotifications)}" />
```

Comment:
Why add parenthesis? Isn't it redundant?

Reply:
Yes, and IMHO it makes the priority explicit.